scipy/interpolate/tests/test_polyint.py

```diff
@@ -368,7 +474,7 @@ def test_large_chebyshev(self):
         # Without capacity scaling or permutation, n=800 fails,
         # With just capacity scaling, n=1097 fails
         # With both capacity scaling and random permutation, n=30000 succeeds
-        n = 800
+        n = 1100
```

Comment:
What is the reason for this change?

Reply:
As indicated in the pre-existing comment, for `n = 800` the test passes even if the random permutation is disabled, so at present it won't catch a regression.